src/UiPath.Workflow/UiPath.Workflow.csproj

```diff
@@ -30,6 +30,9 @@
         <ItemGroup>
             <BuildOutputInPackage Include="$(OutputPath)Microsoft.CodeAnalysis.VisualBasic.Scripting.*"/>
         </ItemGroup>
+		<ItemGroup Condition="'$(TargetFramework)' == 'net6.0' AND '$(Configuration)'=='Debug'">
```

Comment:
Was this missing before?

Reply:
helps debugging with Studio :). No real use in Release mode